packages/@lwc/engine/src/framework/restrictions.ts

```diff
@@ -78,7 +78,7 @@ function portalRestrictionErrorMessage(name: string, type: string) {
 
 function getNodeRestrictionsDescriptors(
     node: Node,
-    options: RestrictionsOptions
+    options: RestrictionsOptions = {}
```

Comment:
up until now, we have been trying to avoid any default value for arguments due to the perf implication in IE11, the transpiled code is very complete (including a loop).

Reply:
The rest parameters are transpiled using a loop when targetting ES5, it's not the case for default parameter: http://www.typescriptlang.org/play/index.html?target=1#code/GYVwdgxgLglg9mABMOcAUBDAXIgzlAJxjAHMAaRAIx3yNMQF5EByAEwFNgMQAbKZgJSIA3gChEExBAS44PdgDoecEpgqUBAblEBfUaJTpmAC3Y9lgzUA